.github/workflows/ci.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node: [10, 11, 12, 13, 14]
+        node: [10, 12, 13, 14]
```

Comment:
maybe better to keep 11 here since #314 already adresses

Reply:
yarn install won't work with node v11 which breaks the CI

Reply:
but tests are still passing on v11 so not a breaking change

Reply:
> yarn install won't work with node v11 which breaks the CI

isn't it supported anymore in 11 ?

Reply:
yup, seems so.